README.md

```diff
@@ -1,2 +1,56 @@
-# secdev-quarantinenetwork
-source code for the Quarantine Network self-service portal
+# SecDev Quarantine Network
+
```

Comment:
Please Add a badge here for Drone:

```
[![Build Status](https://drone.techservices.illinois.edu/api/badges/techservicesillinois/secdev-quarantinenetwork/status.svg)](https://drone.techservices.illinois.edu/techservicesillinois/secdev-quarantinenetwork)
```

Please note that this points to the `master` branch.